go.mod

```diff
@@ -49,12 +49,12 @@ require (
 require (
 	github.com/14rcole/gopopulate v0.0.0-20180821133914-b175b219e774 // indirect
 	github.com/Microsoft/go-winio v0.6.1 // indirect
-	github.com/Microsoft/hcsshim v0.10.0 // indirect
+	github.com/Microsoft/hcsshim v0.11.0 // indirect
 	github.com/VividCortex/ewma v1.2.0 // indirect
 	github.com/acarl005/stripansi v0.0.0-20180116102854-5a71ef0e047d // indirect
 	github.com/asaskevich/govalidator v0.0.0-20230301143203-a9d515a09cc2 // indirect
 	github.com/chzyer/readline v1.5.1 // indirect
-	github.com/containerd/cgroups/v3 v3.0.2 // indirect
+	github.com/containerd/cgroups v1.1.0 // indirect
```

Comment:
@rhatdan @nalind Reading the release notes https://github.com/microsoft/hcsshim/releases/tag/v0.10.0 / https://github.com/microsoft/hcsshim/releases/tag/v0.11.0 , apparently they decided to revert to an earlier codebase. Compare https://github.com/microsoft/hcsshim/issues/1889 .

That subpackage is only used by `c/storage/drivers/windows`. Do we care? AFAICT there was not _one_ commit to that driver, ever since c/storage has existed, adding anything Windows-specific; just updates for driver API changes.

And… whether we care or not, what should we do here? As noted in https://github.com/containers/storage/pull/1704 , this adds bloat, but more importantly downgrades to include a deprecated protobuf implementation.

As a starting point for discussion, I propose rolling `hcsshim` forward instead of backward, let’s say to `v0.12.0-rc.0`, the first tag after `v0.10.0`.

Reply:
That works for me.  I missed this. Will update container/storage tmorrow.